health/notifications/matrix/README.md

```diff
@@ -0,0 +1,42 @@
+# matrix.org - riot.im
+
+Send notifications to matrix network rooms.
```

Comment:
I think we should be a bit more explicit about the services used in this method.

```suggestion
Send notifications to [Matrix](https://matrix.org/) network rooms using the [Riot](https://about.riot.im/) client.
```

Reply:
I haven't found Riot requirement inside the code, so I don't think this change should be applied

Reply:
Fair enough! I think I didn't fully understand the relationship between the two when I first started reviewing. Let's still please capitalize **Matrix**, but you can indeed disregard the other requests.

Reply:
Thanks!. Done.